spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -86,6 +82,85 @@
     expect(page).not_to have_content(adventure.name)
   end
 
+  scenario "authenticated user successfully views list of their adventures" \
+  " by navigating to the adventures index page" do
+    user = FactoryGirl.create(:user)
+    sign_in(user)
+
+    # bucket_list_1
+    bucket_list_1 = FactoryGirl.create(:bucket_list, user_id: user.id)
+    adventure_1 = FactoryGirl.create(
+      :adventure,
+      user_id: user.id
+    )
+    bucket_list_adventure_1 = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure_1`. Did you mean `bucket_list_adventure_2`?